### ISSUE TITLE
[Question] Defender for Cloud support on ARM64-based node pools

### ISSUE THREAD

**Issue by Flasheh** [Azure/AKS#3301]
**Describe scenario**
I am interested in deploying the ARM64-based VMs for some of our node pools, however currently I am blocked because Defender for Cloud does not seem to be supported on this architecture. I am receiving the following error when deploying:

> Enabling Azure Defender on AKS cluster with ARM64 node pools(or adding ARM64 node pool to AKS cluster with Azure Defender enabled) is not allowed.

**Question**
Does anybody know if this functionality is being worked on and if I can track it's progress?

**Comment by dewolfs** [comment]
Hi

Defender for Cloud depends on the [Log Analytics agent](https://learn.microsoft.com/en-us/azure/azure-monitor/agents/log-analytics-agent). Ensure your machines are running one of the supported operating systems for this agent.

Make sure you are running the Azure Monitoring Agent.
Supported OS: Ubuntu 20.04 LTS & Ubuntu 18.04 LTS.

Kr,

**Comment by SabareeshGC** [comment]
Any update on this

**Comment by SabareeshGC** [comment]
@dewolfs based on this it appears arm64 doesnt support Mircosoft Defender and i believe that is what op is getting at https://learn.microsoft.com/en-us/azure/aks/use-multiple-node-pools#add-an-arm64-node-pool
"ARM64 node pools are not supported on Defender-enabled clusters"

**Comment by Flasheh** [comment]
@SabareeshGC Yes that is exactly what I am referring to. The limitation was not documented when I encountered it.

I would really like to use ARM64 node pools, but the client requires enabling Defender on the cluster

**Comment by pcampagnano** [comment]
+1 for this - there are real cost and environmental implications for adding this support.

**Comment by mblaschke-daimlertruck** [comment]
+1

**Comment by Ryangr0** [comment]
Any progress so far? It's kind of bizarre this doesn't "just work"...

**Comment by mayaherskovic** [comment]
Defender for Containers sensor was updated to support ARM 64 nodes in Feb 2024. See release notes in https://learn.microsoft.com/en-us/azure/defender-for-cloud/release-notes-archive#update-new-version-of-defender-sensor-for-defender-for-containers.
This issue should be closed.
